File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/13.cpp
#include <iostream>
using namespace std;

int main(){
	float num1, num2;
	
	cout << "Digite dois numeros: ";
	cin >> num1 >> num2;
	
	if(num1 > num2){
		cout << num1 << " e maior que " << num2 << "\n";
		cout << num2 << " e menor que " << num1;
	} else if(num1 < num2) {
		cout << num2 << " e maior que " << num1 << "\n";
		cout << num1 << " e menor que " << num2;
	} else if(num1 == num2){
		cout << num1 << " igual a " << num2;
	}
	
}

File: 1º Semestre/Programação de Computadores/Lista de Exercicios 2/Python/13.py
"""
13. Elabore um algoritmo que leia dois números e imprima qual é maior, qual é menor, ou se
são iguais
"""

num1 = float(input("Digite o primeiro numero: "))
num2 = float(input("Digite o segundo numero: "))

if num1 > num2:
    print("{:.2f} é maior que {:.2f}.".format(num1, num2))
elif num2 > num1:
    print("{:.2f} é maior que {:.2f}.".format(num2, num1))
else:
    print("{:.2f} e {:.2f} são iguais.".format(num1, num2))